crypto_kem/ledakemlt12/leaktime/sort.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SORT_H
+#define SORT_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#define int32 int32_t
+
+#define int32_MINMAX(a,b) \
+    do { \
+        int32 ab = (b) ^ (a); \
+        int32 c = (b) - (a); \
+        c ^= ab & (c ^ (b)); \
+        c >>= 31; \
+        c &= ab; \
+        (a) ^= c; \
+        (b) ^= c; \
+    } while(0)
```

Comment:
Isn't it more appropriate to move these macros out of the `.h` file and into the `sort.c` file, where they're used?